Documentation Changes for Teams and Workspaces 1.0 (#959)
100mslive/100ms-docs: common/basics.md

-   `Role` A role defines who can a peer see/hear, the quality at which they publish their video, whether they have permissions to publish video/screenshare, mute someone, change someone's role.
-   `Template` A template is a collection of roles, room settings, recording and RTMP settings (if used), that are used by the SDK to decide which geography to connect to, which tracks to return to the client, whether to turn on recording when a room is created, etc. Each room is associated with a template.
-   `Destinations` Destinations is used to save audio/video calls for offline viewing. 100ms supports 2 kinds of recording - SFU recording and Browser recording. Also, `HLS enabled` configuration will allow you to live stream your room over HLS.
-   `RTMP` RTMP streaming is used to live stream your video conferencing apps to platforms like YouTube, Twitch, Facebook, MUX, etc.
-   `Webhooks` Webhook is an HTTP(S) endpoint used for pushing the notifications to your application. It will be invoked by 100ms servers to notify events of your room.

> -   `Workspace` A workspace is an isolated environment which contains account data like templates, rooms, room and session history, etc. You can use workspaces to create differentiated environments like “Production” and “Development” or team-first environments like “Small Classroom” and “Webinar Classroom”.

Reviewer: ```suggestion
You can use workspaces to create differentiated environments like “Production” and “Development” or team-first environments like “Small Classroom” and “Webinar Classroom”.
You can use workspaces to represent isolated environments like “Production” and “Development” and invite team members to a workspace.
```

> -   `Workspace` A workspace is an isolated environment which contains account data like templates, rooms, room and session history, etc. You can use workspaces to create differentiated environments like “Production” and “Development” or team-first environments like “Small Classroom” and “Webinar Classroom”.

Reviewer: ```suggestion
-   `Workspace` A workspace is an isolated environment which contains account data like templates, rooms, room and session history, etc. You can use workspaces to represent environments like “Production” and “Development” and invite team members to a workspace.
```
